index.html

```diff
@@ -1521,6 +1479,55 @@ <h3>Hashlinks</h3>
 
     </section>
 
+    <section>
+      <h2>Zero-Knowledge Proofs</h2>
+      <em>This section is non-normative</em>
+        <p>
+The Verifiable Credentials Data Model is designed to be compatible with a
+variety of existing and emerging digital proof formats. Each
+proof format has benefits and drawbacks. Many proof formats cannot reveal
+selected attribute values in a verifiable credential; they can only reveal all
+(or none).
+        </p>
+        <p>
+Zero-Knowledge Proofs (ZKPs) are a proof format that enables
+privacy-preserving and data-minimization features in
```

Comment:
`privacy-preserving and data-minimization`
-> `privacy-preservation and data-minimization`
or
-> `privacy-preserving and data-minimizing`

Whether `ation` or `ing` is chosen here, text throughout should also be changed to match.

Reply:
Would this work?
```
enables
privacy-preserving data-minimization features . . .
```

Reply:
That would work for this sentence. The rest of the doc should still be checked for similar disagreements.

Reply:
fixed in [5ff637c](https://github.com/w3c/vc-imp-guide/pull/32/commits/5ff637c784e5bec18463221499a55af27f8746e1)